califa/R3BCalifaGeometry.cxx

```diff
@@ -112,6 +119,10 @@ const TVector3& R3BCalifaGeometry::GetAngles(Int_t iD)
     if (cache.count(iD))
         return cache[iD];
 
+    // SOLUTION FOR DOUBLE READING CHANNELS
+    if (iD > fNumCrystals / 2 && iD <= fNumCrystals)
+        iD = iD - fNumCrystals / 2; // for double reading crystals (crystals from 1 to 2432)
+
```

Comment:
NB: changing iD will result in ``cache[originalId]`` not getting updated. This means that we will call the expensive ``R3BCalifaGeometry::GetCrystalVolumePath`` for each hit in the low gain range.